File: Model.Tests/StandingTests.vb
Imports System.Text
Imports Microsoft.VisualStudio.TestTools.UnitTesting

<TestClass()> _
Public Class StandingTests

    <TestMethod()> _
    Public Sub StandingTest()
        ' Arrange
        Dim team = New Team(1)

        ' Act
        Dim result = New Standing(team)

        ' Assert
        Assert.AreSame(team, result.Team, "Unexpected Team")
    End Sub

End Class

File: POFF.Kicker.Tests/StandingTests.cs
using NUnit.Framework;
using POFF.Kicker.Domain;
using POFF.Kicker.Domain.ScoreModes;

namespace POFF.Kicker.Tests;

[TestFixture]
class StandingTests
{
    [Test]
    public void StandingTest()
    {
        // Arrange
        var team = new Team(1);

        // Act
        var result = new Standing(team);

        // Assert
        Assert.That(result.Team, Is.SameAs(team));
    }
}